### DISCUSSION TITLE
Adding ability to pull Calendar event after saving

### DISCUSSION THREAD

**Discussion by wookelvin** [wmcmahan/react-native-calendar-events#374] [Q&A]
Is there an easy way to pull the calendar event back up through the native calendar app after successfully saving the EventID? 

i.e. 
`showCalendarEvent(eventID);`
would pull up original calendar event in either ios/android and be able to edit the original values.

**Comment by wmcmahan** [comment]
@wookelvin No, but this should be easy to accomplish outside of this library. I hesitate to add any UI actions to the api.
